travel_platform/travel_app/urls.py

```diff
@@ -4,4 +4,5 @@
 
 urlpatterns = [
     url(r'^$', views.index, name='index'),
+    url(r'^.*coord/.*$', views.coord, name='coord')
```

Comment:
But GET request contains more data around name of view. The only one function sends this address in request is map. What should be changed?